### DISCUSSION TITLE
Aggregate like sum

### DISCUSSION THREAD

**Discussion by Tarang** [supabase/supabase#766] [Questions]
Is it possible to do aggregation with supabase, for example with `sum`

```
  .select(`
	      sum(amount)
   `)
```

**Comment by steve-chavez** [comment]
Yes, but not directly through supabase-js. See https://github.com/supabase/supabase/discussions/432 and https://github.com/supabase/supabase/discussions/626#discussioncomment-328540.

**Reply by Tarang** [reply]
@steve-chavez Do views have realtime updates if a row is added?

**Reply by kiwicopple** [reply]
The don't have realtime updates @Tarang - only tables will receive the updates
